src/movement/test/control_system.test

```diff
@@ -0,0 +1,14 @@
+<launch>
+    <test test-name="test_control_system" pkg="robosub" type="test_control_system" />
+
+    <node name="control" pkg="robosub" type="control"/>
+
+    <rosparam command="load" file="$(find robosub)/param/cobalt.yaml" />
+
+    <include file="$(find robosub_simulator)/launch/gazebo.launch">
```

Comment:
This requires the robosub_simulator package to be present. Is there an alternative if it doesn't? If not there could end up being circular dependencies between the two.